google_maps/google_maps.py

```diff
@@ -41,12 +45,69 @@ class GoogleMapsXBlock(XBlock):
         scope=Scope.settings
     )
 
-    iframe_height = Integer(
-        display_name=_('Map height'),
-        default=450,
+    place = Dict(
+        display_name=_('Information about the place being searched'),
+        default={
+            'id': 'ChIJCwHM0mF344kRAWE3Rmv-RcY',  # ExtensionEngine, Boston headquarters
+            'name': '196 Broadway, Cambridge, MA 02139, USA'
+        },
         scope=Scope.settings
     )
 
+    @property
+    def api_key(self):
+        """
+        Returns the Google Maps API key from settings service.
+
+        The API key should be set in both lms/cms env.json files inside XBLOCK_SETTINGS.
+
+        Example:
+            "XBLOCK_SETTINGS": {
+                "GoogleMapsXBlock": {
+                    "API_KEY": "YOUR API KEY GOES HERE"
+                }
+            },
+        """
+        return SettingsService().get_settings_bucket(self).get('API_KEY', '')
```

Comment:
Yes, that was the point of blocking this in the first place.
You have to fix the bug in the edx-platform.